pcsx2/gui/Panels/VideoPanel.cpp

```diff
@@ -30,7 +30,7 @@ Panels::FramelimiterPanel::FramelimiterPanel( wxWindow* parent )
 	: BaseApplicableConfigPanel_SpecificConfig( parent )
 {
 	m_check_LimiterDisable = new pxCheckBox( this, _("Disable Framelimiting"),
-		_("Useful for running benchmarks. Toggle this option in-game by pressing F4.") );
+		_("Useful for running benchmarks. Toggle this option in-game by pressing F4. It basically uncaps FPS to unlimited") );
```

Comment:
Isn't the option name itself self explanatory?

Reply:
Perhaps, i just felt it was even more obvious what it does. It feels like a double negative, maybe a bit too nitpicky? I don't see the harm for expanding this part.

Reply:
I'd probably change that last sentence to just "Uncaps FPS.". Basically is one of these words that can almost always be deleted without consequence, and I think unlimited is already implied in "uncaps".

Seems to me like it should be at the beginning, not the end, too. In fact, maybe:
"Uncaps FPS. Toggle in-game by pressing F4. (Useful for running benchmarks.)" ?

Reply:
Ah yes fair point, i was going to change that and forgot about it.